assignments/prototypes.js

```diff
@@ -9,11 +9,21 @@
 */
   
 /*
-  === GameObject ===
+  === GameObject ===constructor
   * createdAt
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
+function GameObject(attrs) {
```

Comment:
GameObject function should have `createdAt`, `dimensions`, and `destroy()` and You added name, weapons etc.